relay/src/cli.rs

```diff
@@ -110,7 +110,7 @@ pub fn extract_config_env_vars() -> OverridableConfig {
         id: env::var("RELAY_ID").ok(),
         public_key: env::var("RELAY_PUBLIC_KEY").ok(),
         secret_key: env::var("RELAY_SECRET_KEY").ok(),
-        outcome_source: None, //already extracted in params
```

Comment:
Why this change?

Reply:
youre quick! just an artefact from accidentally editing master and failing to revert properly, ive fixed it